SCF-Session-8/zooManagement/MammalZone.java

```diff
@@ -0,0 +1,19 @@
+package zooManagement;
+
+/**
+ * This class will add the Mammal zone into Zoo
+ * @author Ishita_Mathur
+ */
+public class MammalZone extends Zone
+{
```

Comment:
You can remove this class `MammalZone` as it does not have any extra feature for zone. Zone can be identified by `AnimalFamily`.